.github/workflows/test-runner-on-perf.yml

```
@@ -14,10 +14,19 @@ jobs:
   test-runner-perf:
     runs-on: ubuntu-latest
     steps:
-      - name: "Checkout jade-data-repo develop branch"
+      - name: "Fetch latest semantic version from data-repo dev"
+        id: "read_property"
+        run: |
+          CURRENT_VERSION=$(curl -s -X GET "https://jade-perf.datarepo-perf.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "Current Version: $CURRENT_VERSION"
+          echo "::set-output name=CURRENT_SEMVER::$CURRENT_VERSION" 
+          LATEST_VERSION=$(curl -s -X GET "https://jade.datarepo-dev.broadinstitute.org/configuration" -H "accept: application/json" | jq -r '.semVer|rtrimstr("-SNAPSHOT")')
+          echo "Latest Version: $LATEST_VERSION"
+          echo "::set-output name=LATEST_SEMVER::$LATEST_VERSION"  
```

Is there a reason to use `SEMVER` here over `VERSION`?
```suggestion
          echo "::set-output name=LATEST_VERSION::$LATEST_VERSION"  
```

If we ever change what we use for version again then this variable name might need to be changed again. Using VERSION instead could avoid the need to change the name. It's a really minor thing but I think about the name choice as being a way to hide (or expose) implementation. The code that uses this variable doesn't care if the version is a semantic version or a githash (or something else), it just needs a way to name things and compare things based on version.

---

I like that idea  - thanks!